packages/typescript-eslint-parser/src/parser-options.ts

```diff
@@ -0,0 +1,12 @@
+export interface ParserOptions {
+  useJSXTextNode?: boolean;
+  loc?: true;
+  range?: true;
+  tokens?: true;
+  filePath?: string;
+  sourceType?: 'script' | 'module';
+  ecmaVersion?: number;
+  ecmaFeatures?: {
+    globalReturn?: boolean;
```

Comment:
https://github.com/typescript-eslint/typescript-eslint/pull/29/files#diff-536465f1f982b433f6fba58e560438b2R49

Reply:
I believe this is currently a top-level key, but I do think we should deprecate that and move it into `ecmaFeatures` (will solve https://github.com/typescript-eslint/typescript-eslint/issues/24).

Since this will be published as a new package, maybe now is a good time to make that change? 

Relevant PR from the old repo here: https://github.com/eslint/typescript-eslint-parser/pull/595

Reply:
Agreed, I've added a note about deprecation, we can re-implement 595 in a follow up